dotnet/src/VectorData/Pinecone/PineconeFilterTranslator.cs

```diff
@@ -23,13 +23,21 @@ internal class PineconeFilterTranslator
     private Extensions.VectorData.ProviderServices.CollectionModel _model = null!;
     private ParameterExpression _recordParameter = null!;
 
-    internal Metadata Translate(LambdaExpression lambdaExpression, Extensions.VectorData.ProviderServices.CollectionModel model)
+    internal Metadata? Translate(LambdaExpression lambdaExpression, Extensions.VectorData.ProviderServices.CollectionModel model)
     {
         this._model = model;
 
         Debug.Assert(lambdaExpression.Parameters.Count == 1);
         this._recordParameter = lambdaExpression.Parameters[0];
 
+        // Weaviate doesn't seem to have a native way of expressing "always true" filters; since this scenario is important for fetching
```

Comment:
Just to make sure: the new tests have discovered a bug and this is a fix for it? (it looks like it could be also part of some other PR)

Reply:
Yeah, this is just some fixup to #13294 - the Pinecone tests weren't working properly when I was working on that. So this is an unrelated change.